cerebrium/other-topics/faster-cold-starts.mdx

```diff
@@ -1,15 +1,38 @@
 ---
-title: Decrease Model Loading Time
-description: Decrease the time it takes to load your model from storage into GPU
+title: Faster Cold Starts
+description: Decrease the time it takes start your application
 ---
 
```

Comment:
Can we add to this section that increasing core counts can improve pull-through times for large images as the download is parallelised?